### DISCUSSION TITLE
Redis method's description

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#3009] [Help] [answered]
I am using Redis for storing some data to optimize the performance. I searched in Adonisjs Docs about deleting(Since my use case is updating the data by deleting the old one) key-value pair data from the Redis but I didn't found anything yet. I need the following methods:
1) Method to delete data from Redis based on key
2) Method to reset Redis datastore
And also it would be helpful if you provide the documentation of all the methods available in Redis. @thetutlage 
Any guidance will be helpful.

**Comment by RomainLanz** [comment, marked as answer]
Hey @feniljariwala82! 👋 

As said in the documentation, we are using [`ioredis`](https://github.com/luin/ioredis) internally. Have you checked their documentation?

For example, you can delete a record based on its key with the `del()` method.

**Reply by feniljariwala82** [reply]
Thank you for the help'
